packages/acceptance-tests/pkg-tests-specs/sources/commands/set/version.test.ts

```diff
@@ -101,7 +101,7 @@ describe(`Commands`, () => {
         await xfs.writeFilePromise(ppath.join(projectDir, Filename.lockfile), ``);
 
         await run(`set`, `version`, `self`, {cwd: projectDir});
-        await check(projectDir, {corepackVersion: /[0-9]+\./, usePath: true});
+        await check(projectDir, {corepackVersion: /[0-9]+\./, usePath: true}, ppath.join(path, `.yarn/releases`));
```

Comment:
In the tests where the yarnPath is set outside of the project, simply omitting the releasesDir parameter (in the new version of the `check` function) wouldn't work.